tests/integration/test_events.py

```diff
@@ -1263,3 +1338,11 @@ def cleanup(
         if queue_url:
             sqs_client = sqs_client or aws_stack.create_external_boto_client("sqs")
             sqs_client.delete_queue(QueueUrl=queue_url)
+        if log_group_name:
+            logs_client = logs_client or aws_stack.create_external_boto_client("logs")
+            log_streams = logs_client.describe_log_streams(logGroupName=log_group_name)
+            for log_stream in log_streams["logStreams"]:
+                logs_client.delete_log_stream(
+                    logGroupName=log_group_name, logStreamName=log_stream["logStreamName"]
+                )
+            logs_client.delete_log_group(logGroupName=log_group_name)
```

Comment:
This might actually be a good time to clean up the `test_events.py` file and bring it up to date in terms of testing patterns in a separate PR if you're interested :+1:

Reply:
small remark to the cleanup function (don't need to fix this): any exception in here will cause all cleanups afterwards to be skipped. This is mostly a problem when dealing with tests against AWS (e.g. when debugging). You'll be creating a lot of orphaned resources this way.